[FTI-3281] add an option `config.disable_penalty` to rate-limiting-advanced plugin (#4755)
Kong/docs.konghq.com: app/_hub/kong-inc/rate-limiting-advanced/_index.md

        group override, but does not clear the list of consumer groups.
        You can then flip `enforce_consumer_groups` to `true` to re-enforce the
        groups.
    - name: disable_penalty
      minimum_version: "3.1.x"
      required: false
      default: false
      value_in_examples: null
      datatype: boolean
      description: |

>         If set to `true`, then won't count the denied requests (status = `429`),
>         while by default all requests including denied ones will be counted.
>         This parameter only has an effect on `sliding` window_type.

Reviewer: ```suggestion
        If set to `true`, this doesn't count denied requests (status = `429`). If set to `false`, all requests, including denied ones, are counted. This parameter only affects the `sliding` window_type.
```
